src/cheri/insns/auipc_32bit.adoc

```diff
@@ -26,7 +26,7 @@ Increment the address of the AUIPC instruction's <<pcc>> by the 32-bit offset, t
 include::rep_range_check.adoc[]
 
 // CHERIoT uses an 11-bit shift due to a different capability format
-NOTE: Future CHERI extensions that use an alternative capability format may choose to redefine the handling of the immediate operand for this instruction in {cheri_cap_mode_name}.
+//NOTE: Future CHERI extensions that use an alternative capability format may choose to redefine the handling of the immediate operand for this instruction in {cheri_cap_mode_name}.
```

Comment:
This is going to be a problem for CHERIoT, unless we are indeed a different base, but see discussion [over there](../../issues/581)

Reply:
yes - I think this one is insurmountable - the only way to redefine instructions is to have a new base, and changing AUIPC and the capability encoding format does exactly that. That's my understanding.

Reply:
done - I've added a note that AUIPC eeds tobe in the extension with the encoding. not in the base